src/Types/WhiteLabel.ts

```diff
@@ -0,0 +1,2 @@
+export const ALL_VALID_WHITE_LABELS = ['co', 'nc', 'co_energy_calculator'] as const;
+export type WhiteLabel = (typeof ALL_VALID_WHITE_LABELS)[number];
```

Comment:
@CalebPena What do you think about using `export type WhiteLabel = 'co' | 'nc' | 'co_energy_calculator'` instead for readability? If other values need to be added to the `ALL_VALID_WHITE_LABELS` type it's super quick to add those to the `WhiteLabel` type as well.